api/tests/routes/backoffice/bank_account_test.py

```diff
@@ -55,6 +55,7 @@ def test_get_bank_account(self, authenticated_client):
         assert f"BIC : {bank_account.bic} " in response_text
         assert "Statut dossier DMS Adage :" not in response_text
         assert "Pas de dossier DMS CB" in response_text
+        assert "État du compte bancaire :" in response_text
```

Comment:
Ici tu testes seulement que l'intitulé est présent, or il est en dur dans le code, donc rien d'intéressant n'est testé.
Vérifie au moins la valeur affichée en face, par exemple : 
```suggestion
        assert "État du compte bancaire : Accepté" in response_text
```
ici ou dans un test qui contient effectivement un compte dans l'état testé.